org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/adwindow/ADWindowToolbar.java

```diff
@@ -898,6 +898,8 @@ public void updateToolbarAccess() {
 		ADWindow adwindow = ADWindow.findADWindow(this);
 
 		dynamicDisplay();
+		pressedLogic();
```

Comment:
I think it is better to call both logic method at the end of dynamicDisplay instead. With this, we can make both *logic method private and don't have to hunt for places where dynamicDisplay is call.

Reply:
@hengsin  Please note that I made changes as suggested.